File: 2-semestre/uri-online-judge/java/1099.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);

        int n = entrada.nextInt();
        int i = 0;

        while (i < n) {
            int x = entrada.nextInt();
            int y = entrada.nextInt();
            int soma = 0;
            
            if (x < y) {
                for (int aux = x+1; aux < y; aux++) {
                    if (aux % 2 != 0) soma += aux;
                }
            } else {
                for (int aux = y+1; aux < x; aux++) {
                    if (aux % 2 != 0) soma += aux;
                }
            }

            System.out.printf("%d\n", soma);

            i++;
        }
    }
}

File: 2-semestre/uri-online-judge/c/1099.c
#include <stdio.h>
 
int main() {
    int n, i = 0, x, y, aux;

    scanf("%d", &n);

    while (i < n) {
        scanf("%d", &x);
        scanf("%d", &y);
        int soma = 0;
        
        if (x < y) {
            for (aux = x+1; aux < y; aux++) {
                if (aux % 2 != 0) soma += aux;
            }
        } else {
            for (aux = y+1; aux < x; aux++) {
                if (aux % 2 != 0) soma += aux;
            }
        }

        printf("%d\n", soma);

        i++;
    }

    return 0;
}
